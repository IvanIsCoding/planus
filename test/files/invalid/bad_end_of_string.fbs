table Foo {
  x: string = "hello;
}