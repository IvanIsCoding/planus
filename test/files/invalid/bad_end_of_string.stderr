[0m[1m[38;5;9merror[0m[1m: Unexpected end of string[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/bad_end_of_string.fbs:3:2
  [0m[34mâ”‚[0m
[0m[34m3[0m [0m[34mâ”‚[0m }
  [0m[34mâ”‚[0m  [0m[31m^[0m

Error: Bailing because of errors
